packages/core/src/redux/slices/remoteParticipants.ts

```diff
@@ -211,10 +211,7 @@ export const remoteParticipantsSlice = createSlice({
 
             return {
                 ...state,
-                remoteParticipants: clients
-                    .filter((c) => c.id !== selfId)
-                    .filter((c) => !NON_PERSON_ROLES.includes(c.role.roleName))
-                    .map((c) => createRemoteParticipant(c)),
+                remoteParticipants: clients.filter((c) => c.id !== selfId).map((c) => createRemoteParticipant(c)),
```

Comment:
Isn't this a breaking change? Some clients probably need to tweak their app to exclude non-person clients from their video grid, if this gets merged. 🤔

Reply:
Ah nevermind, as I see we still have the filter in the `selectRemoteParticipants = createSelector(selectRemoteClients, (clients) => {}` fn on line 307 👍

Reply:
Yea. That was the one I noticed as well. So I _think_ it wont break anything.